lib/Ohmbrewer_Heating_Element.cpp

```diff
@@ -43,15 +45,16 @@ Ohmbrewer::HeatingElement::~HeatingElement() {
 }
 
 
-/**
+/**TODO remove overwriting?
```

Comment:
Yes, if HeatingElement always does what Relay does for a method, just leave the method out of HeatingElement. We can always add it back in, if needed.